Tests/Shared/extension/QueueMockExtensions.swift

```diff
@@ -0,0 +1,11 @@
+@testable import CioTracking
+import Common
+import Foundation
+
+public extension QueueMock {
+    var deviceTokensDeleted: [String] {
+        addTaskReceivedInvocations.map {
```

Comment:
could use a comment, from what i gather you are filtering the list to non-empty values maybe?